pkg/apitype/stacks.go

```diff
@@ -53,6 +56,11 @@ type EncryptValueRequest struct {
 	Plaintext []byte `json:"plaintext"`
 }
 
+type CurrentUpdateStatus struct {
+	CurrentUpdateStarted int64 `json:"currentUpdateStarted,omitempty"`
```

Comment:
Any type or function that is exported from a Go package (i.e. starts with a capital letter) should have a doc comment explaining what it is. Even if the type name is super-clear, we should still add a doc comment for it. When it comes to exported _fields_ of types however, use your best judgement. Err on the side of documenting type fields if it isn't clear what format the data is, or what it is referring to.

So in this case, we should have a comment on `type CurrentUpdateStatus`. But omitting a comment from `CurrentUpdateStarted` and `CurrentUpdateAuthor` is probably OK.